### PR TITLE
bathsheba fixes for release

### PR DESCRIPTION
**Pull request by minorwm**
# READY FOR REVIEW

# Summary

This is all the fixes that are caught in QA before the release

# Needed By (Date)
- Release

# Criticality
- Client is eagerly awaiting this.

# Steps to Test
## Update  SoE Helper Page
See this for sub-task tickets 19 - 24 here: https://stanfordits.atlassian.net/browse/SOE-3025
- Adding to the stanford_soe_helper_page feature for the top banner fields
- Adding the colored bar to the caption of the top banner
- Fixing css for the Stanford Page Top Banner for the bottom right and left as well as top right and left positioning/responsive design
- Adding css for the infotext class
- Adding css for the superhead class
- Adding css for the testimonial block
- Fixing span wrapping divs in collection view
- Turning cache on for the collection view

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Folks to notify
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: scss/mixins/_soe_breakpoints.scss [hunk 1]
```diff
@@ -3,7 +3,9 @@
 // MAX-WIDTH BREAKPOINTS
 
 @mixin breakpoint-max($point) {
-  @if $point == large {
+  @if $point == x-large {
+    @media (max-width: 1250px) { @content; }
+  }@else if $point == large {
```
Comment (sherakama) [inline scss/mixins/_soe_breakpoints.scss]: Not sure why the linter missed this but there should be a space after the curly brace and the else statement.

File: scss/mixins/_soe_breakpoints.scss [hunk 2]
```diff
@@ -3,7 +3,9 @@
...
     @media (max-width: 1200px) { @content; }
   }
   @else if $point == medium {
```
Comment (sherakama) [inline scss/mixins/_soe_breakpoints.scss]: Also, it would be nice to fix this up as well even though it is older code.

**Comment by sherakama** [comment]
The settings for the linter can be found here: https://github.com/SU-SOE/stanford_soe_helper/blob/7.x-2.x/.codeclimate.yml#L18

File: scss/mixins/_soe_breakpoints.scss (hunk 1 above)
Comment (minorwm) [reply scss/mixins/_soe_breakpoints.scss]: @sherakama fixed

File: modules/stanford_soe_helper_page/stanford_soe_helper_page.info [hunk 3]
```diff
@@ -29,32 +29,32 @@ features[ctools][] = views:views_default:3.0
...
 features[field_instance][] = node-stanford_page-field_s_banner_caption_horizonta
 features[field_instance][] = node-stanford_page-field_s_banner_caption_vertical
 features[field_instance][] = node-stanford_page-field_s_banner_image
 features[field_instance][] = node-stanford_page-field_s_banner_link
 features[field_instance][] = node-stanford_page-field_s_top_banner_caption_horiz
 features[field_instance][] = node-stanford_page-field_s_top_banner_caption_vert
+features[field_instance][] = node-stanford_page-field_s_top_banner_color
 features[views_view][] = stanford_page_banner_caption
 features[views_view][] = stanford_page_top_banner
-features_exclude[field_base][field_s_banner_accent_color] = field_s_banner_accent_color
```
Comment (jbickar) [inline modules/stanford_soe_helper_page/stanford_soe_helper_page.info:52]: @sherakama they don't want to remove these `features_exclude` lines, do they?

File: modules/stanford_soe_helper_page/stanford_soe_helper_page.info [hunk 4]
```diff
@@ -29,32 +29,32 @@ features[ctools][] = views:views_default:3.0
...
 features[field_base][] = field_s_top_banner_caption_horiz
 features[field_base][] = field_s_top_banner_caption_vert
+features[field_base][] = field_s_top_banner_color
 features[field_group][] = group_s_page_banner|node|stanford_page|form
 features[field_group][] = group_s_page_top_banner|node|stanford_page|form
 features[field_group][] = group_s_postcard_content|node|stanford_page|full
 features[field_group][] = group_s_postcard_image|node|stanford_page|full
 features[field_group][] = group_s_postcard|node|stanford_page|full
 features[field_instance][] = node-stanford_page-field_s_apachesolr_index
+features[field_instance][] = node-stanford_page-field_s_banner_accent_color
```
Comment (jbickar) [inline modules/stanford_soe_helper_page/stanford_soe_helper_page.info:46]: @sherakama I don't think they want to add these in. I think there will be a conflict with `field_instance`s and `field_base`s in other Features.

Comment (sherakama) [reply modules/stanford_soe_helper_page/stanford_soe_helper_page.info:46]: @jbickar @minorwm @cjwest 

This depends; If this module is adding these fields to the stanford-page type then this is ok. If this module is tracking fields that are in other modules then it is not ok. 

It looks like this is adding to the stanford-page type and not duplicating values. Please confirm that this assumption is correct and that the .info file has a dependency on stanford_page.

File: modules/stanford_soe_helper_page/stanford_soe_helper_page.info (hunk 3 above)
Comment (sherakama) [reply modules/stanford_soe_helper_page/stanford_soe_helper_page.info:52]: They do if they want to track them. See previous comment.

File: modules/stanford_soe_helper_page/stanford_soe_helper_page.info (hunk 4 above)
Comment (jbickar) [reply modules/stanford_soe_helper_page/stanford_soe_helper_page.info:46]: OK, thanks.

File: modules/stanford_soe_helper_page/stanford_soe_helper_page.info (hunk 3 above)
Comment (jbickar) [reply modules/stanford_soe_helper_page/stanford_soe_helper_page.info:52]: OK, per further discussion and investigation: I was confused how the `features_exclude` lines got in here in the first place. There was WIP on these components that got excluded the last time the Feature was exported. So now they _do_ want to include that work, and it's independent of `stanford_page`.

**Comment by cjwest** [comment]
Hi all,


Yes, we want to explicitly exclude that taxonomy. Because this taxonomy (and others such as department) is used across several content types, we add the taxonomy programmatically.

See for example:
https://github.com/SU-SOE/stanford_soe_helper/blob/7.x-2.x/modules/stanford_soe_helper_page/stanford_soe_helper_page.install#L37


________________________________

**Pull request merged**